com.unity.render-pipelines.core/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
As discussed with Tim, please use ## [7.5.0] - 2020-06-08 instead of ## [Unreleased].  for this particular package, thanks